docs/setup/docker.asciidoc

```diff
@@ -124,14 +124,11 @@ images:
 `server.name`:: `kibana`
 `server.host`:: `"0"`
 `elasticsearch.url`:: `http://elasticsearch:9200`
-
-In the {xpack} image, the following additional defaults are also set:
-
-[horizontal]
-`elasticsearch.username`:: `elastic`
-`elasticsearch.password`:: `changeme`
 `xpack.monitoring.ui.container.elasticsearch.enabled`:: `true`
 
+NOTE: The setting `xpack.monitoring.ui.container.elasticsearch.enabled` is not
+defined in the `-oss` image.
```

Comment:
Perhaps add a link to the following page for more information about configuring monitoring?: https://www.elastic.co/guide/en/kibana/master/monitoring-xpack-kibana.html

Reply:
Monitoring works out of the box with the default values. There's no configuration needed with the full image. Happy to add the link, but is it needed with the defaults working?

Reply:
You're right, it's not necessary. Things won't work until collection is enabled in Elasticsearch, but the link is optional here.